azure-pipelines.yml

```diff
@@ -63,6 +63,28 @@ jobs:
         artifactName: 'pros_cli-$(Build.BuildNumber)-macos-$(buildArch)'
         targetPath: 'out'
 
+- job: macOS M1
+  timeoutInMinutes: 30
+  dependsOn: UpdateBuildNumber
+  strategy:
+    maxParallel: 2
+    matrix:
+      x64:
+        buildArch: x64
```

Comment:
not sure about this one, might want to update the above to amd64 and this to arm64